RW2: enable zero_is_bad for all files

pedrocr opened the pull request:
As discussed in issue #120 we can just enable zero_is_bad for all RW2 files as the format always uses this.

pedrocr committed:
RW2: enable zero_is_bad for all files

pedrocr commented:
Is there something here you don't like? I left the flag in the code so it's easy to disable it in the future if for some reason a camera shows up where this is not true (doing a zero_is_not_bad flag for example).

klauspost commented:
No, I just haven't had the time to test it out, sorry for the delay.

klauspost commented:
Could you make it so, that it is enabled by default, but you still have the option to explicitly disable it?

That way, if we should encounter issues with a specific camera we can still disable it.

pedrocr committed:
RW2: add a zero_is_not_bad for possible future use
In case we find a camera where zero_is_bad should not be used we have a flag to disable it.

pedrocr commented:
I wasn't worried about the delay at all, not trying to rush you. I just asked because you had merged later PRs so I wondered if there was anything wrong with this one. I've added an extra commit to add the zero_is_not_bad flag in case we need it.

klauspost commented:
ok, let's try it out :)
